ci/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ endif()
 find_package(bacio 2.4.0 REQUIRED)
 find_package(sp 2.3.3 REQUIRED)
 find_package(w3emc 2.9.2 REQUIRED)
+find_package(PIO REQUIRED)
```

Comment:
Why is PIO required?

Reply:
I had added a test for module_write_netcdf that was failing because pio wasn't linked, and this took care of it. I don't think it's critical for this PR but I believe it'll be needed for future tests.

Reply:
It only just occurred to me that this may be an issue with FindESMF.cmake, i.e., it may not be passing the right linker flags. I'll look into it further.

Reply:
It's not necessary currently, so I'm removing it for now and I can circle back to it if it becomes an issue for future tests.